scripts/xjoin-config/elasticsearch-sink.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "xjoin.inventory.es",
+    "config": {
+      "type.name": "_doc",
+      "connector.class": "io.confluent.connect.elasticsearch.ElasticsearchSinkConnector",
+      "behavior.on.null.values": "delete",
+      "errors.log.include.messages": "true",
+      "transforms": "valueToKey, extractKey, expandJSON, expandPRSJSON, deleteIf, flattenList, flattenListString, flattenPRS, renameTopic",
+      "max.retries": "8",
```

Comment:
The `max.retries` can be removed as default `5` should work?

Reply:
Done